Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -18,6 +18,8 @@ let crlf: StaticString = "\r\n"
 let headerSeparator: StaticString = ": "
 
 private let connectionUtf8 = "connection".utf8
+private let closeUtf8 = "close".utf8
```

Comment:
I think these global variables do nothing (in the good case) and cause a slowdown if the compiler isn't able to optimise it. Just put them where you use them

Reply:
@weissi hmm I would have thought its actually better as `.utf8` should only be called once ? Maybe I should make these static ?

Reply:
@normanmaurer `.utf8` is a function that needs to be inlined. If you want to cache the bytes straight, you'll need to do `Array("close".utf8)`